internal/lox_files_test.go

```
@@ -0,0 +1,126 @@
+		}
+
+		// Check if the last byte is a newline
+		if buf[0] == '\n' {
+			t.Errorf("File %s ends with a newline", file)
+		}
+	}
+}
+
+func TestLineLength(t *testing.T) {
+	const maxLineLength = 51
+	os.Chdir("../test_programs")
```

golangci-lint (1.64.8)

Error return value of `os.Chdir` is not checked

(errcheck)